x-pack/plugins/maps/public/inspector/vector_tile_adapter/components/get_tile_request.ts

```diff
@@ -35,11 +35,16 @@ export function getTileRequest(tileRequest: TileRequest): { path?: string; body?
   }
   const geometryFieldName = searchParams.get('geometryFieldName') as string;
 
+  const hasLabels = searchParams.has('hasLabels')
```

Comment:
Try adding a label to an MVT layer, then remove the label. The inspector still shows `with_labels: true`. I think this happens because `canSkipSourceUpdate` is returning true even though we are changing the search parameters.

Reply:
resolved